launcher/src/main/kotlin/com/cognifide/gradle/aem/launcher/BuildScaffolder.kt

```diff
@@ -9,9 +10,19 @@ class BuildScaffolder(private val launcher: Launcher) {
         saveProperties()
         saveSettings()
         saveRootBuildScript()
-        saveEnvBuildScript()
+        if (Regex("6.5.*").matches(propertyAemVersion())) {
```

Comment:
I would go differently here...

val aemVersion
```
if (aemVersion.isNotBlank()) {
    val scaffolder = if (aemVersion == "cloud") EnvCloudScaffolder(launcher) else EnvOnPremScaffolder(launcher)
    scaffolder.scaffold()
}
```

because your regex will fail on planned AEM 6.6 :) and upcoming...